G/Graphviz/build_tarballs.jl

```diff
@@ -58,6 +58,8 @@ if [[ "${target}" == *-mingw* ]]; then
     # Cover up the traces of the hack
     rm ${prefix}/include/regex.h
 fi
+
+${bindir}/dot -c
```

Comment:
We can't generally run programs built for the target platform

Reply:
https://github.com/JuliaPackaging/Yggdrasil/blob/master/G/Graphviz/bundled/patches/0002-do-not-build-dot-config.patch

Reply:
Ah, shucks. That makes sense now that I think of it. I guess this responsibility gets shifted to the JLL user then? It would be cool if there were a way to inject something into the `__init__`.

Reply:
Actually you can already: https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.build_tarballs

But I wonder if something like https://github.com/JuliaGraphics/Gtk.jl/blob/master/src/Gtk.jl#L84 makes more sense. @staticfloat any advice on what would be the best strategy here?

Reply:
Yeah, I would do something like what Gtk does, although I probably would not use an Artifact for it, I'd use a Scratch space once that lands in 1.6